infra-templates/gandi-dns/config.yml

```diff
@@ -0,0 +1,5 @@
+name: GANDI DNS
+description: |
+  Rancher External DNS service powered by Gandi.net
+version: v0.1.0
+category: External DNS
```

Comment:
Please add the label as shown here https://github.com/rancher/community-catalog/blob/master/infra-templates/digitalocean-dns/config.yml#L6-L7